activity_calendar/templates/activity_calendar/activity_page_new_slot.html

```diff
@@ -0,0 +1,67 @@
+{% extends "activity_calendar/activity_page_base.html" %}
+{% load render_form %}
+{% load static %}
+{% load calendar_tags %}
+
+
+{% block XXX %}
+    <a href="{{ main_activity_url }}" class="btn btn-outline-dark btn-sm my-2">
+        <img
+                src="{% static 'octicons/arrow-left.svg' %}"
+                alt=""
+                height="12"
+                width="12"
+        />
+        Back to activity details
+    </a>
+    {{ block.super }}
+{% endblock %}
+
+{% block sign-up-status %}
+    {% if is_subscribed %}
+        <div class="alert alert-success">
+            <div class="row">
+                <div class="col-12">
+                    <div class="p-1">
+                        {% if subscribed_slots.count == 1 %}
+                            You are subscribed to slot '{{ subscribed_slots.first.activity_slot.title }}'
+                        {% else %}
+                            You are subscribed to slots:
+                            {% for slot_entry in subscribed_slots %}
+                                {% if forloop.last %}and {% endif %}
+                                '{{ slot_entry.activity_slot.title }}'{% if not forloop.last %},{% endif %}
+                            {% endfor %}
+                        {% endif %}
+                    </div>
+                </div>
+            </div>
+        </div>
+    {% else %}
+        <div class="alert bg-light border border-danger">
+            <div class="row">
+                <div class="col-12">
+                        You are not registered for this activity
+                </div>
+                <div class="col-12">
+                    Go back to main page to register on a slot{% if activity.slot_creation == "CREATION_USER" %}
+                    or create a slot below
+                    {% else %}. {% endif %}
+                </div>
+            </div>
+        </div>
+    {% endif %}
+    <a href="{{ main_activity_url }}" class="btn btn-secondary greenify btn-block my-2">
+        Back to main page
+    </a>
+{% endblock sign-up-status %}
+
+
+{% block additional-contents %}
+    <hr>
+    <h2>Create new slot</h2>
+
+    <div class="wideContentFrame">
+        {% render_form id="create_slot_form" method="post" submit_class="btn btn-success btn-block" submit_text="Create slot" %}
+    </div>
+
+{% endblock %}
```

Comment:
Please add a trailing newline.

_Looks at repo_

... why is there no editorconfig file in Squire yet. BRB.